Scripts/import-logs.ps1

```diff
@@ -0,0 +1,110 @@
+#########powershell script to upload log files into s3 bucket###########
+
+#s3Bucket path to save the log files
+param (
+    [string] $version,
+    [string] $ReleaseId
+)
+$Lansaversion = $version
+$PipelineReleaseId = $ReleaseId
+$s3BasePath = 'lansa-us-east-1/db-regression-test/logs' 
+$region = 'us-east-1'
+ 
+#Total Summary for all Dbs
+$totalSummaryFile = 'C:\Program Files (x86)\LANSA\Verifier_Total_Summary.txt'
+$totalSummaryPath = Join-Path $Lansaversion $PipelineReleaseId
+$totalSummaryKey = $totalSummaryPath + '/Verifier_Total_Summary.txt'
+Write-S3Object -BucketName $s3BasePath -Key $totalSummaryKey -Content $totalSummaryFile 
+
+
+################################ list of databases ################################################################
+$databases = New-Object -TypeName 'System.Collections.ArrayList';
```

Comment:
The style here is good. And you need to be adding in databases based on the Database Types passed in.

Reply:
Hi @robe070 I have seen your comments, will address those and get back